src/CoreFoundation/CFMutableString.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Microsoft Corporation
+
+#if !COREBUILD
+
+using System;
+using System.Runtime.InteropServices;
+
+using Foundation;
+using ObjCRuntime;
+
+namespace CoreFoundation {
+	
+	public class CFMutableString : CFString {
```

Comment:
`sealed`?

Reply:
why ? it's not common in our bindings (for direct, public bindings) and the linker can do that optimization.

Reply:
It seems that subclassing CFMutableString is something nobody should ever do, and besides it's always possible unseal it later if we want to.

Reply:
Swift does not seal it and I can't see any positive outcome out of sealing it. At best it does nothing (linker will seal it), at worse someone will need that (for good or bad reasons) and it will takes weeks/months to get a fix out.